lib/localio/localizable_writer.rb

```diff
@@ -1,5 +1,6 @@
 require 'localio/writers/android_writer'
 require 'localio/writers/ios_writer'
+require 'localio/writers/oeg_writer'
```

Comment:
I expect Robert and I will require a separate file, since our outputs shouldn't be in javascript. It's possible we could use the same file and modify it for our outputs. But I think it's possible it makes more sense to have a separate hub writer, because the ticket suggests only including hub data, which is probably easiest to do by having a separate writer.